examples/fabric-bridge-app/linux/main.cpp

```diff
@@ -59,8 +61,48 @@ constexpr uint16_t kPollIntervalMs = 100;
 constexpr uint16_t kRetryIntervalS = 3;
 #endif
 
+uint16_t gFabricAdminServerPort = 33001;
+uint16_t gLocalServerPort       = 33002;
+
 BridgedDeviceBasicInformationImpl gBridgedDeviceBasicInformationAttributes;
 
+constexpr uint16_t kOptionFabricAdminServerPortNumber = 0xFF01;
+constexpr uint16_t kOptionLocalServerPortNumber       = 0xFF02;
+
+ArgParser::OptionDef sProgramCustomOptionDefs[] = {
+    { "fabric-admin-server-port", ArgParser::kArgumentRequired, kOptionFabricAdminServerPortNumber },
```

Comment:
If the argument is required do we need to define a default? or is there a why for this to remain optional?

Reply:
Is this just saying that if optionals are provided we need to provide both?

Reply:
This tells the parser that the argument for the option is required, i.e.:  `--option REQUIRED_ARGUMENT`